region/norfair/west/Ice Beam Snake Room.json

```diff
@@ -950,6 +998,46 @@
       ]
     },
     {
+      "link": [2, 3],
+      "name": "R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"enemyKill": {"enemies": [["Fune"]]}},
+        {"or": [
+          {"and": [
+            "h_heatedCrystalFlashForReserveEnergy",
+            {"heatFrames": 280}
+          ]},
+          {"and": [
+            "Morph",
+            "h_heatProof",
+            "h_RModeCanRefillReserves",
+            {"enemyKill": {"enemies": [["Fune", "Fune"], ["Fune"]]}},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 40}}
+          ]}
+        ]},
+        {"or": [
+          {"canShineCharge": {"usedTiles": 12, "openEnd": 0}},
+          {"and": [
+            {"doorUnlockedAtNode": 3},
+            {"canShineCharge": {"usedTiles": 13, "openEnd": 0}}
+          ]}
+        ]},
+        "h_heatTriggerRModeSparkInterrupt",
+        {"heatFrames": 10}
+      ],
+      "unlocksDoors": [
+        {"nodeId": 3, "types": ["ammo"], "requires": []}
+      ],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Kill the Funes first, then the Sovas. Shinecharge at the bottom and use heat damage to interrupt."
```

Comment:
Why that order?  It sounds wrong since it may increase the chances of ammo drops.